implementations/rust/ockam/ockam_command/src/forwarder/create.rs

```diff
@@ -108,13 +90,11 @@ async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, CreateCommand)) -> R
         } else {
             cmd.forwarder_name.clone()
         };
-        let body = CreateForwarder::new(
-            ma,
-            Some(alias),
-            at_rust_node,
-            pa,
-            CredentialExchangeMode::Oneway,
-        );
+        let body = if Some(Project::CODE) == cmd.at.first().map(|p| p.code()) {
+            CreateForwarder::at_project(ma, Some(alias), cmd.cloud_opts.route())
```

Comment:
Should we error if `cmd.authorized` was provided as well?

Reply:
Done in https://github.com/build-trust/ockam/pull/3528/commits/08d78aed923f83858d6996fcc98fa4a09f71a9a8